plugins/modules/route53_health_check.py

```diff
@@ -340,16 +343,22 @@ def main():
         check_id = existing_check.Id
         existing_config = to_health_check(existing_check.HealthCheckConfig)
 
-    if state_in == 'present':
+    if state_in in ('present', 'enabled', 'disabled'):
+        state_new = 0
+        if state_in == 'disabled':
+            state_new = 1
+        state_old = 0
```

Comment:
I chose this route because I was having issues converting from boolean to int in order to match the XML output from Boto2. This logic will either not be necessary, or simplified, when the code gets converted to Boto3.

Reply:
Ah, ok thanks that makes sense - I'll admit I didn't dig into the boto2 docs much.  :)